nikola/data/themes/base/templates/feeds_translations_helper.tmpl

```diff
@@ -1,91 +1,97 @@
 ## -*- coding: utf-8 -*-
 
-## Handles both feeds and translations
-<%def name="head(classification=None)">
-    % if rss_link:
+<%def name="_append_language(language)">${ " (" + language + ")" if len(translations) > 1 else "" }</%def>
```

Comment:
The `_proper` was to avoid having a nested `if` in a one-liner :)

I've now refactored it to avoid one-line macros in 00b2a808e2d96866323926b27ded41c30a8db348 and 1bb186b764b8ecae6c1d8f665ca5a8bb507c4e4f.